curation_portal/models.py

```diff
@@ -45,7 +45,7 @@ class Variant(models.Model):
         max_length=6, choices=[("GRCh37", "GRCh37"), ("GRCh38", "GRCh38")], default="GRCh37"
     )
     variant_id = models.CharField(max_length=1000)
-    sample_ids = ArrayField(models.CharField(max_length=1000, blank=True), null=True, blank=True)
+    sampleIDs = ArrayField(models.CharField(max_length=1000, blank=True), null=True, blank=True)
```

Comment:
Any reason to change this to camelCase? I'd really prefer to stick to snake_case in the django app for consistency.

Reply:
Oh whoops I just changed it to match the example JSON - we should change it on @EddieLF's end

Reply:
Oh ok, I'll let him know!